sources/packages/backend/apps/workers/src/services/application/application.service.e2e.skip-spec.ts

```diff
@@ -77,7 +77,7 @@ describe.skip("ApplicationService", () => {
     jest.spyOn(msfaaNumberService, "createMSFAANumber");
   });
 
-  describe("associateMSFAANumber", () => {
+  describe.skip("associateMSFAANumber", () => {
```

Comment:
This test is not "active". Just renamed it right now to be outside the worker's E2E tests execution.
It must be fixed. It was moved here during previous PRs.